test/ui/BandCompTest.js

```diff
@@ -0,0 +1,131 @@
+const webdriver = require('selenium-webdriver');
+const chrome = require('selenium-webdriver/chrome');
+const chromedriver = require('chromedriver');
+const By = webdriver.By
+chrome.setDefaultService(new chrome.ServiceBuilder(chromedriver.path).build());
+var map = webdriver.promise.map;
+var chai = require('chai');
+var should = require('chai').should();
+var axios = require('axios');
+var MockAdapter = require('axios-mock-adapter');
+const JobService = require('../../app/service/JobService');
+const expect = chai.expect;
+
+describe('Page band-comp should include competency and competency information', () => {
+    it('should return band competencies', async () => {
+
+        var driver = await new webdriver.Builder()
+            .setChromeOptions(new chrome.Options().headless())
+            .withCapabilities(webdriver.Capabilities.chrome())
+            .build();
+
+        //launch job-roles page
+        driver.get('http://localhost:3000/job-roles');
```

Comment:
this will work but wouldn't allow you to run the tests against the app when it is deployed in AWS, have a think about how you can pass the base URL in when you're running the UI tests